posts/2022-07-20-liberty-tools-eclipse.adoc

```diff
@@ -0,0 +1,93 @@
+author_picture: https://avatars3.githubusercontent.com/awisniew90
+author_github: https://github.com/awisniew90
+seo-title: Introducing Liberty Tools (for the Eclipse IDE)
+seo-description: A next generation plugin that brings the power of Open Liberty into your favorite IDE
+blog_description: "A next generation plugin that brings the power of Open Liberty into your favorite IDE"
+open-graph-image: https://openliberty.io/img/blog/liberty-tools-actions.png
+---
+= Introducing Liberty Tools (for the Eclipse IDE)
+Adam Wisniewski <https://github.com/awisniew90>
+
+Liberty Tools (for Eclipse) provides a new experience for developing applications with Open Liberty. This early release offers support for editing Liberty configuration (server.xml) and developing MicroProfile applications, a Liberty Dashboard for organizing your projects, and "dev mode" from right within the IDE. 
+
+Check out a quick demo:
+
+< Replace link with new demo link >
+++++
+<iframe width="560" height="315" align="center" src="https://www.youtube.com/embed/e_oIInKFtHw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+++++
+
+== Liberty Dashboard
+
+The Liberty Dashboard keeps all of your Maven and Gradle projects organized and is automatically populated with projects that are configured to run with Open Liberty. With a few clicks, you can start and stop your app, run tests and view test reports. 
+
+== Dev Mode
+
+Liberty Dev mode automatically picks up your code changes and deploys them to your running app all without having to restart the server. With Liberty Tools, your app starts in dev mode automatically giving you all of the benefits of faster development. 
+
+== MicroProfile and Liberty config support
+
+Configuring your Liberty server is now simpler than ever with server config auto-complete, validation, and "hover over" descriptions.
+
+Additionally, you can quickly develop MicroProfile applications with automatic code blocks, auto-complete, and real time MP syntax validation. 
+
+
+== Try it out
+
+=== Installation
+
+Install the plugin using the instructions found link:https://github.com/OpenLiberty/liberty-tools-eclipse/blob/main/docs/installation/installation.md[here]
+
+=== Using the Dashboard
+
+Once installed, you will see the Open Liberty UFO logo in the top tray. Click on that to open the Liberty Dashboard tab:
+
+[.img_border_light]
+image::/img/blog/liberty-tools-dash-icon.png[Liberty Dashboard Icon,width=70%,align="center"]
+
+The Dashboard will automatically populate with any Maven and Gradle projects in your workspace that are configured to run with Open Liberty (server.xml in the default location). If you do not have any apps in your current workspace, try importing one like the Open Liberty "Getting Started" app link:https://github.com/openliberty/guide-getting-started[here]. Once imported, refresh the Dashboard tab by clicking the refresh icon in the top right. 
```

Comment:
```suggestion
The Dashboard automatically populates with any Maven and Gradle projects in your workspace that are configured to run with Open Liberty, with the `server.xml ` file in the default location. If you do not have any apps in your current workspace, try importing one, like the link:https://github.com/openliberty/guide-getting-started[Open Liberty Starter app]. Once you import an app, refresh the Dashboard tab by clicking the refresh icon.
```

Reply:
also want to avoid "top right"- if needed you could add markup to the refresh icon to indicate the location